runtime/lua/vim/filetype/detect.lua

```diff
@@ -1652,10 +1657,10 @@ local function match_from_text(contents, path)
   return cvs_diff(path, contents)
 end
 
-M.match_contents = function(contents, path)
+M.match_contents = function(contents, path, dispatch_extension)
```

Comment:
I realize this wasn't done in this PR, but can you change this signature to

```suggestion
function M.match_contents(contents, path, dispatch_extension)
```

That is the preferred style for Neovim, I missed this when it was first merged.

Reply:
Done